WcaOnRails/spec/models/team_spec.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Team do
+  before(:all) do
+    # Names were generated from FactoryBot.create :user
+    @names = ["Larry Ullrich", "Jeffry Kiehn", "Colby Nader", "Lakia Cremin", "Shanae Price", "Ka Hermiston", "Milton Pfeffer", "Jolie Cartwright", "Alana Monahan", "Elden Cruickshank"]
+    @users = []
+    @names.each do |name|
+      @users.append(FactoryBot.create(:user, name: name))
+    end
+  end
+
+  before do
+    @wrt_team = FactoryBot.create :team, friendly_id: 'wrt'
+    FactoryBot.create :team_member, user_id: @users[0].id, team_id: @wrt_team.id, start_date: Time.now - 10.months, team_leader: true, updated_at: Time.now - 10.months
+    FactoryBot.create :team_member, user_id: @users[1].id, team_id: @wrt_team.id, start_date: Time.now - 10.months, team_senior_member: true, updated_at: Time.now - 10.months
+    FactoryBot.create :team_member, user_id: @users[2].id, team_id: @wrt_team.id, start_date: Time.now - 10.months, team_senior_member: true, updated_at: Time.now - 10.months
+    FactoryBot.create :team_member, user_id: @users[3].id, team_id: @wrt_team.id, start_date: Time.now - 10.months, updated_at: Time.now - 10.months
+    FactoryBot.create :team_member, user_id: @users[4].id, team_id: @wrt_team.id, start_date: Time.now - 10.months, updated_at: Time.now - 10.months
+    FactoryBot.create :team_member, user_id: @users[5].id, team_id: @wrt_team.id, start_date: Time.now - 10.months, updated_at: Time.now - 10.months
+    @wrt_team.reload
+  end
+
+  it "Added 2 new members" do
+    FactoryBot.create :team_member, user_id: @users[6].id, team_id: @wrt_team.id, start_date: Time.now - 4.days
+    FactoryBot.create :team_member, user_id: @users[7].id, team_id: @wrt_team.id, start_date: Time.now - 4.days
+
+    expected_output = [
+      "Changes in WCA Results Team",
+      "",
+      "New Members",
+      "Jolie Cartwright",
+      "Milton Pfeffer",
```

Comment:
Refer to `users[n].name` here instead of expecting one exact name.

Reply:
Done.